games-emulation/pcsx2/pcsx2-1.4.0-r1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+MY_PV="${PV/_/-}"
+
+inherit cmake-utils l10n multilib toolchain-funcs wxwidgets
+
+DESCRIPTION="A PlayStation 2 emulator"
+HOMEPAGE="http://www.pcsx2.net"
+SRC_URI="https://github.com/PCSX2/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	app-arch/bzip2[abi_x86_32(-)]
+	app-arch/xz-utils[abi_x86_32(-)]
+	dev-libs/libaio[abi_x86_32(-)]
+	media-libs/alsa-lib[abi_x86_32(-)]
+	media-libs/libpng:=[abi_x86_32(-)]
+	media-libs/libsdl[abi_x86_32(-),joystick,sound]
+	media-libs/libsoundtouch[abi_x86_32(-)]
+	media-libs/portaudio[abi_x86_32(-)]
+	>=sys-libs/zlib-1.2.4[abi_x86_32(-)]
+	virtual/jpeg:62[abi_x86_32(-)]
+	virtual/opengl[abi_x86_32(-)]
+	x11-libs/gtk+:2[abi_x86_32(-)]
+	x11-libs/libICE[abi_x86_32(-)]
+	x11-libs/libX11[abi_x86_32(-)]
+	x11-libs/libXext[abi_x86_32(-)]
+	x11-libs/wxGTK:3.0[abi_x86_32(-),X]
+"
+# Ensure no incompatible headers from eselect-opengl are installed, bug #510730
+DEPEND="${RDEPEND}
+	>=app-eselect/eselect-opengl-1.3.1
+	dev-cpp/pngpp
+	>=dev-cpp/sparsehash-1.5
+"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+PATCHES="${FILESDIR}"/"${P}"-gcc5.patch
```

Comment:
It does allow strings, but using an array is *highly encouraged*. This was common to all my reviews and is also mentioned in https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/